scripts/check-status

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
```

Comment:
Just because it's not actually required and it prevents us having to update all the scripts where this is used if we change it to some other language. I guess we could do the whole symlink thing like we are in the plugins dir  if this is the more correct way to do it https://github.com/feedhenry/nagios-container/tree/master/plugins/default?